build.gradle

```diff
@@ -279,6 +295,78 @@ subprojects {
       exceptionFormat = testExceptionFormat
     }
     logTestStdout.rehydrate(delegate, owner, this)()
+
+    afterTest { TestDescriptor desc , TestResult result ->
+      if (TestResult.ResultType.FAILURE == result.resultType) {
+        logger.warn("Found failing test $desc.parent - $desc.name, trying again in next turn of test execution")
```

Comment:
Should we log this only if `shouldFilterFlakyTests == true`

Reply:
Yes, probably.